src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -27,10 +72,11 @@ const convertInclude = ({ href, url, type, ...rest }) => {
   return {
     type,
     model: {
-      // `url` here should be replaced with `href` once mozart routes have been created. /*TODO: Create issue for this */
-      href: url,
-      ...rest,
+      href,
+      // `includeUrls[type]` here should be replaced with `href` once mozart routes have been created. /*TODO: Create issue for this */
```

Comment:
```suggestion
      // `includeUrls[includeType]` here should be replaced with `href` once mozart routes have been created. /*TODO: Create issue for this */
```
I thought an issue was created for this?

Reply:
yeah an issue was created, will edit the comment